core/src/main/scala/org/apache/spark/h2o/backends/SharedBackendConf.scala

```diff
@@ -164,15 +169,29 @@ trait SharedBackendConf {
 
   def setNodeBasePort(port: Int) = set(PROP_NODE_PORT_BASE._1, port.toString)
 
+  def setHadoopConf(path: String) = set(PROP_HADOOP_CONF._1, path)
+
+  def setHadoopConf(conf: Configuration): H2OConf = {
+    val hadoopConfigTempFile: File = File.createTempFile("hadoop_config_all", ".xml")
```

Comment:
Shouldn't we delete the file after the driver terminates? 
```hadoopConfigTempFile.deleteOnExit()```

Reply:
Yes, we should. Changed. Gave it a try after this change once again on the cluster - works. Thank you.